README.md

```diff
@@ -1,2 +1,14 @@
 # Hello-Microverse
 Hello Microverse Project - Module 1
+
+## Built With
+
+- HTML
+- CSS
+- Linter
+
+## Authors
```

Comment:
- [OPTIONANL] Please update your Readme, currently some headings are missing such as  

```
##  Getting Started 
```

It will be nice to add explicit steps to clone and run your project locally, as this will be of help to non-technical users. Please add descriptive steps from cloning your app to seeing the results displayed on the browser.. This includes `git clone URL` , `cd project_name` etc

- I would also recommend to add the following: 

```
## 🤝 Contributing

Contributions, issues, and feature requests are welcome!

Feel free to check the [issues page](../../issues/).

## Show your support

Give a ⭐️ if you like this project!

## Acknowledgments

- Microverse
```

Reply:
@BenMukebo Thank you so much for your, Approval and optional suggestions. I have made some of the changes already in this repo. Rest will follow in my future projects.

Reply:
@BenMukebo One small query do I need to close the pull request after the merging? Please guide me on this, as I am new to the process.

Reply:
Hi @RitobrotoMukherjee,

Don't worry all depend on you, There's no problem in deleting branches that have been merged in. All the commits are still available in the history, and even in the GitHub interface, they will still show up, if the branch was fully merged into master before deleting it, there should not be any need for the branch any longer.

To do that you can watch this video on you tube about [How to Auto Delete Branch after Pull Request merge](https://www.youtube.com/watch?v=-gEKL-yinKQ)

Hope this help